content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -21,6 +21,9 @@ The `extraArgs` field consist of `key: value` pairs. To override a flag for a co
 
 1.  Add the appropriate fields to your configuration.
 2.  Add the flags to override to the field.
+3.  Update the configuration by `kubeadm upgrade apply --config config.yml` 
```

Comment:
```suggestion
3.  Update the configuration by running `kubeadm upgrade apply --config config.yml` 
```
What do you think @kigawas?

Reply:
-1 on including `kubeadm upgrade apply --config config.yml`.
applying upgrade for the sake of modifying a flag is possible, but not recommended.

there are plans to support simpler customizations eventually.

Reply:
> -1 on including `kubeadm upgrade apply --config config.yml`.
> applying upgrade for the sake of modifying a flag is possible, but not recommended.
> 
> there are plans to support simpler customizations eventually.

Thanks for pointing out.

Can you make a recommendation of modifying flags at the current 14.2 version?

Reply:
with 1.14 and 1.15 you must modify the `/etc/kubernetes/manifests/` YAML files manually, but these changes will be lost after upgrade. to make the changes persist you must also modify the `kubeadm-config` ConfigMap object.

`kubeadm upgrade apply --config config.yml` solves this problem but it also performs all the steps of an upgrade which might not be desired by some users.

Reply:
Thanks, so a user may need to modify files under `/etc/kubernetes/manifests/` and both `kubeadm-config` configmap?

I've done some search, it looks like this point is seldom mentioned. Shall we document it here?

Reply:
no, this should not be documented here.